lib/src/util/js_component.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:over_react/over_react.dart';
+import 'package:react/react_client/component_factory.dart';
+
+/// Creates a Dart component factory that wraps a ReactJS [factoryProxy].
```

Comment:
#nit Might be nice to add a note that we're going to follow up with more in-depth documentation 
```suggestion
/// Creates a Dart component factory that wraps a ReactJS [factoryProxy].
///
/// More in-depth documentation for wrapping JS components is coming soon.
```